src/PC/Chebyshev.jl

```diff
@@ -50,20 +52,30 @@ mutable struct Chebyshev <: AbstractPreconditioner
     # data never changed
     operator::Operator
     eigenvaluebounds::Array{Float64,1}
+    chebyshevtype::ChebyshevType.ChebyType
 
     # data empty until assembled
     eigenvalueestimates::Array{Float64,1}
     operatordiagonalinverse::AbstractArray{Float64}
 
     # inner constructor
-    Chebyshev(operator::Operator) = new(operator, [0.0, 0.1, 0.0, 1.0])
+    Chebyshev(operator::Operator) = new(operator, [0.0, 0.1, 0.0, 1.0], ChebyshevType.first)
```

Comment:
I think that the constructor should optionally accept a type with the default value of `ChebyshevType.first`